zap/src/main/java/org/zaproxy/zap/extension/ascan/PolicyAllCategoryPanel.java

```diff
@@ -535,7 +535,8 @@ public void validateParam(Object obj) throws Exception {
 
         } else if (!newName.equals(currentName)) {
```

Comment:
This should be changed to be case insensitive as well, otherwise it prevents changing the case of the policy itself.

Reply:
Or not, this would require further validation if a policy with same case already exists in case sensitive file systems.